### DISCUSSION TITLE
How To Stop Error "Parsing error: Using the export keyword between a decorator and a class is not allowed. Please use `export @dec class` instead."

### DISCUSSION THREAD

**Discussion by LogicRoom** [codesandbox/codesandbox-client#5287] [General] [answered]
We have the error "Parsing error: Using the export keyword between a decorator and a class is not allowed. Please use `export @dec class` instead." here;

https://codesandbox.io/s/final-project-mobx6-inversify17-jfle1?file=/src/Components/LeftPresenter.js

Not sure how to stop this warning? Do we

1) do something with prettier
2) do something with eslint
3) do something with the jsconfig

Any help much appreciated, had a quick go but couldn't disable it!

**Comment by SaraVieira** [comment, marked as answer]
Hey!

I am so sorry you are having issues with our eslint.

At this moment there is no way to change the configuration or disable eslint and that is something we are working on

I did find a couple of issues related to the error you had: https://stackoverflow.com/questions/53483066/babel-7-using-the-export-keyword-between-a-decorator-and-a-class-is-not-allowed

And exporting it in a different line did make the error go away

http://localhost:3000/s/standards-mobx6-inversify17-forked-3gnki?file=/src/Components/LeftPresenter.js

Sorry there is no better way
